Firefox/webextension/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "content_scripts": [ {
```

Comment:
We need the background script (and can remove that once we release the Firefox non-embedded WebExtension)